webrtc/coverage/set-session-description.txt

```diff
@@ -0,0 +1,242 @@
+Coverage Report is based on the following editor draft:
+https://w3c.github.io/webrtc-pc/archives/20170605/webrtc.html
+
+4.4.1.6 Set the RTCSessionSessionDescription
+
+  [Trivial]
+  1.  Let p be a new promise.
+
+  [Trivial]
+  2.  In parallel, start the process to apply description as described in [JSEP]
+      (section 5.5. and section 5.6.).
+
+    [Trivial]
+    1.  If the process to apply description fails for any reason, then user agent
+        MUST queue a task that runs the following steps:
+
+      [RTCPeerConnection-setLocalDescription]
+      [RTCPeerConnection-setRemoteDescription]
+      1.  If connection's [[IsClosed]] slot is true, then abort these steps.
+
+      [RTCPeerConnection-setLocalDescription]
+      [RTCPeerConnection-setRemoteDescription]
+      2.  If the description's type is invalid for the current signaling state of
+          connection as described in [JSEP] (section 5.5. and section 5.6.), then
+          reject p with a newly created InvalidStateError and abort these steps.
```

Comment:
This is not the contents of 2.1.2 that appear in the link you gave. This is step 2.1.2: 


> If elements of the SDP were modified, then reject p with a newly created InvalidModificationError and abort these steps.

![](https://i.gyazo.com/b23af7d19f4e9f31de3cc4742cc047e9.png)